api/handlers.go

```diff
@@ -413,6 +413,13 @@ func (api *API) userLimitsGET(_ *database.User, w http.ResponseWriter, req *http
 			TierID:     u.Tier,
 			TierLimits: database.UserLimits[u.Tier],
 		}
+		// If the quota is exceeded we should keep the user's tier but report
+		// anonymous-level speeds.
+		if u.QuotaExceeded {
```

Comment:
This endpoint is meant to be accessed by tools such as nginx. If the user wants to know why they have the speeds they do, they would use https://account.siasky.net/api/user where they would get a response like this:
```json
{
    "email": "<removed>",
    "sub": "<removed>",
    "tier": 2,
    "createdAt": "2021-03-18T09:57:58.146Z",
    "migratedAt": "2021-12-15T19:06:55.93Z",
    "subscribedUntil": "2022-02-18T12:56:34Z",
    "subscriptionStatus": "active",
    "subscriptionCancelAt": "1970-01-01T00:00:00Z",
    "subscriptionCancelAtPeriodEnd": false,
    "stripeCustomerId": "<removed>",
    "quotaExceeded": false,
    "emailConfirmed": true
}
```
This will give them their tier and whether their quota is exceeded.